feeder/src/vote.ts

```diff
@@ -295,33 +295,69 @@ interface VoteArgs {
 }
 
 export async function vote(args: VoteArgs): Promise<void> {
-  const client = new LCDClient({
-    URL: args.lcdAddress,
+  const maxLCDIndex = args.lcdAddress.length - 1
+  let currentLCDIndex = 0
+
+  let client = new LCDClient({
+    URL: args.lcdAddress[currentLCDIndex],
     chainID: args.chainID,
     gasPrices: args.gasPrices,
   })
   const rawKey: RawKey = await initKey(args.keyPath, args.password)
   const valAddrs = args.validator || [rawKey.valAddress]
   const voterAddr = rawKey.accAddress
-  const wallet = new Wallet(client, rawKey)
+  let wallet = new Wallet(client, rawKey)
+  let switchLCD = false
 
   while (true) {
     const startTime = Date.now()
 
-    await processVote(client, wallet, args.source, valAddrs, voterAddr).catch((err) => {
-      if (err.isAxiosError && err.response) {
-        console.error(err.message, err.response.data)
-      } else {
-        console.error(err.message)
+    if (switchLCD) {
+      switchLCD = false
+      if (++currentLCDIndex > maxLCDIndex) {
+        currentLCDIndex = 0
       }
+      client = new LCDClient({
+        URL: args.lcdAddress[currentLCDIndex],
```

Comment:
Shouldn't `args.lcdAddress` be of type `string[]` instead of `string`?

Reply:
It' actually a string. I changed the type in VoteArgs.